index.html

```diff
@@ -0,0 +1,180 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href='http://fonts.googleapis.com/css?family=Lato:400,700' rel='stylesheet' type='text/css'>
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <div class="top-page">
+        <a href=""><img src="images/facebook-icon.svg" alt="facebook"></a>
+        <a href=""><img src="images/twitter-icon.svg"></a>
+        <a href="">Logout</a>
+        <a href="">Mypage</a>
+        <a href="">Korean</a>
+    </div>
+    <header>
+        <div class="hamburger">
+            <img src="images/hamburger-icon.svg" alt="menu">
+        </div>
+        <div class="top-right">
+            <div class="mo">
+                <a href="index.html"><img src="images/main-logo.svg" alt="logo"></a>
+            </div>
+        </div>
+        <nav class="nav">
+            <a href="about.html">About</a>
+            <a href="">Program</a>
+            <a href="">Join</a>
+            <a href="">Sponsor</a>
+            <a href="">News</a>
+            <div class="campaign">
+                <a href="#">CC Campaign</a>
+            </div>
+        </nav>
+    </header>
+    <nav class="navformob">
+        <div class="menu-close">
+            <img src="images/menu-cross-icon-1.svg" alt="close-menu">
+        </div>
+        <ul>
+            <li><a href="index.html">Home</a></li>
+            <li><a href="about.html">About</a></li>
+            <li> <a href="#">Sponsor</a></li>
+            <li><a href="#">News</a></li>
+        </ul>
+    </nav>
+    <section class="home-page">
+        <div class="home">
+            <h3>"Hello! Sharing World"</h3>
+            <h1 class="head-title">CREATIVE COMMONS</br>GLOBAL SUMMIT</br>2015</h1>
+        </div>
+        <div class="paragraph">
+            <p>A joyful celebration bellieving in the value of opennes and sharing, creating <br> A posative change with people from over 80 countries is taking place in October, in Korea  A posative change with people from over 80 countries is taking place in October, in Korea</p>
+        </div>
+        <div class="hom-foot">
+            <h3>2015.10.15(THU) ~ 16(FRI)</h3>
+            <h4>@ National Museum of Korea, Art Center Nabi and More</h4>
+        </div>
+    </section>
+    <section class="main-program">
+        <div class="container">
```

Comment:
- [ ] [REQUIRED] Good job so far! Please add correct `hover` effect to each card and also add correct `style` to your `"SEE THE WHOLE PROGRAM"` button like a [Behance Design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015)🙏🏼 

![image](https://user-images.githubusercontent.com/63915024/225583247-ab06d7bc-17f8-4e00-bfb4-1a2ac69622c8.png)